crt_portal/crt_portal/settings.py

```diff
@@ -336,9 +353,6 @@
     ('css', 'compressor_toolkit.precompilers.SCSSCompiler'),
 )
 
-# would like to add this before public release
-COMPRESS_ENABLED = False
```

Comment:
By default this is set to whatever DEBUG is, so omitting this line for convenience

Reply:
👍 , Noting ref from docs: The default value for `COMPRESS_ENABLED` is the _opposite_ of `DEBUG`

https://django-compressor.readthedocs.io/en/stable/settings/#django.conf.settings.COMPRESS_ENABLED